spec/Powertrain/Battery.vspec

```diff
@@ -185,12 +187,12 @@ Charging.Timer.Time:
   datatype: uint32
   type: actuator
   unit: s
-  description: Time value for next charging-related action (Unix timestamp, seconds).
+  description: Time value for next charging-related action (Unix timestamp, seconds). 0 if no timer has been set.
 
 Charging.Timer.Mode:
   datatype: string
   type: actuator
-  enum: [ 'starttime', 'endtime', 'depaturetime' ]
+  enum: [ 'inactive', 'starttime', 'endtime', 'departuretime' ]
```

Comment:
Here I assume that a specific element to indicate that timer-based charging is deactivated could make sense if you want to cancel planned charging, rather than setting Charging.Timer.Time to 0.

I corrected spelling for departuretime as well, but do not know if it makes sense to keep it as long as we do not have any "departuretime" signal in VSS